### ISSUE TITLE
No support for python 3.10

### ISSUE THREAD

**Issue by mlg9000** [pymedusa/Medusa#10065]
Medusa does not run under python 3.10.  "AttributeError: module 'collections' has no attribute 'MutableMapping'"

**Comment by BenjV** [comment]
As of python 3.3 this statement was depreciated.
`from collections import Mapping`

As of python 3.9  it was removed and so it is also not available in python 3.10
The **A**bstract **B**ase Classes for **C**ontainers should be used

That code should be changed in:
`from collections.abc import Mapping`

Or if backwards compatibility is important this should be used:
```
try:
    from collections.abc import Mapping
except ImportError:
    from collections import Mapping
```

**Comment by medariox** [comment]
Fixed with https://github.com/pymedusa/Medusa/pull/10068 
Will be available with the upcoming version.
